crates/bevy_pbr/src/meshlet/gpu_scene.rs

```diff
@@ -104,6 +116,39 @@ pub fn extract_meshlet_meshes(
             .instance_uniforms
             .get_mut()
             .push(MeshUniform::new(&transforms, None));
+        if let Some(render_layers) = render_layers {
```

Comment:
I'm not quite getting the logic, why is there two different branches for with/without render layers?